tests/ui/codegen/mismatched-data-layouts.stderr

```diff
@@ -0,0 +1,4 @@
+error: data-layout for target `mismatched-data-layout-7814813422914914169`, `normalized data layout`, differs from LLVM target's `x86_64-unknown-none-gnu` default layout, `normalized data layout`
```

Comment:
Where does the `-7814813422914914169` part of the target name come from?

Reply:
https://github.com/rust-lang/rust/blob/e35a56d96f7d9d4422f2b7b00bf0bf282b2ec782/compiler/rustc_target/src/spec/mod.rs#L3466-L3482